src/main/resources/logback-spring.xml

```diff
@@ -15,7 +15,7 @@
 
   <logger name="eu.europeana" level="WARN"/>
   <logger name="eu.europeana.metis.sandbox" level="INFO"/>
-  <logger name="org.springframework" level="ERROR"/>
+  <logger name="org.springframework" level="WARN"/>
```

Comment:
After doing troubleshooting of messages going to dead letter queue, I realize this should be at least in WARN level